astro/src/content/docs/extend/examples/5-minute-intro/5-minute-fastpath.mdx

```diff
@@ -57,48 +56,13 @@ Steps similar to these will be used for integrating with any identity provider.
 
 You are following the FastPath installation guide, so you'll use the FastPath method to install FusionAuth.
 
-<FastPathInstall />
-
-Once the Fast Path installer completes, you will see the below output.
-
-```sh title="Fast Path Install Complete"
-Downloading zip packages
-######################################################################## 100.0%
-######################################################################## 100.0%
-Installing packages
-Install is complete. Time for tacos.
-
- 1. To start FusionAuth run the following command
-    /Users/bpontarelli/dev/example/fusionauth/bin/startup.sh
-
- 2. To begin, access FusionAuth by opening a browser to http://localhost:9011
-
- 3. If you're looking for documentation, open your browser and navigate to https://fusionauth.io/docs
-
-Thank you have a nice day.
-```
+<DownloadWidget includeDocker={false}/>
```

Comment:
you pass in `includeDocker` but don't check it in `DownloadWidget.astro`. Is that intentional?

Reply:
The widget has a ton of logic for that variable. Am I missing something?

Reply:
Ah, sorry! DownloadWidget didn't get expanded in the PR screen and I missed it.